__tests__/app.test.js

```diff
@@ -158,7 +158,74 @@ describe('GET /api/reviews', () => {
     })
 })
 
-
+describe('GET /api/reviews/:id/comments', () => {
+    test('200: should have correct length', () => {
+        return request(app)
+        .get('/api/reviews/2/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body
+            expect(comments.length).toBe(3)
+        })
+    })
+    test('200: should have correct keys/values', () => {
+        return request(app)
+        .get('/api/reviews/2/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body
+            expect(comments.length).toBe(3)
```

Comment:
As this expect is the same as the previous test, feel free to combine them together (e.g. 'should return all reviews with correct keys/values')